actions/st2_prep_release_for_st2.sh

```diff
@@ -151,6 +156,48 @@ if [ "${IS_DEV_VERSION}" = "false" ]; then
             fi
         fi
     done
+    for REQUIREMENT_FILE in "${LDAP_REQUIREMENT_FILES[@]}"
+    do
+        echo "Setting ldap branch version in: ${REQUIREMENT_FILE}"
+
+        if [[ ! -e "${REQUIREMENT_FILE}" ]]; then
+            >&2 echo "ERROR: Requirement file ${REQUIREMENT_FILE} does not exist."
+            exit 1
+        fi
+
+        VERSION_STR_MATCH=`grep "git+https://github.com/StackStorm/st2-auth-ldap.git@${BRANCH}#egg=st2-auth-ldap" ${REQUIREMENT_FILE}`
+        if [[ -z "${VERSION_STR_MATCH}" ]]; then
+            echo "Setting ldap version branch in ${REQUIREMENT_FILE} to ${BRANCH}..."
+            sed -i -E "s/^git\+https:\/\/github.com\/StackStorm\/st2-auth-ldap\.git@(.*?)st2-auth-ldap$/git\+https:\/\/github.com\/StackStorm\/st2-auth-ldap\.git@${BRANCH}#egg=st2-auth-ldap/" ${REQUIREMENT_FILE}
+
+            VERSION_STR_MATCH=`grep "st2-auth-ldap.git@${BRANCH}" ${REQUIREMENT_FILE} || true`
+            if [[ -z "${VERSION_STR_MATCH}" ]]; then
+                >&2 echo "ERROR: Unable to update the ldap version in >${REQUIREMENT_FILE}."
+                exit 1
+            fi
+        fi
+    done
```

Comment:
NAK. Multiple issues with this.

1. Use the `FORK` variable instead of hardcoding `StackStorm` so this respects the existing parameters.
2. Don't match on `#egg=`, see above.
3. Instead of checking for the `BRANCH` before updating it, I would remove the `if` check and "blindly" do the search-and-replace with `sed`. If that ends up not changing anything, then the script will detect that farther down and won't try to commit or push. This keeps the script more simple, easier to understand and easier to edit if/when we switch to releasing `st2-auth-ldap` and `st2-rbac-backend` as their own Python packages and change these lines.
4. Not sure what value the post-`sed` check has.
5. The `>` seems weird in an error message, and makes copying and pasting that filename more difficult. I would remove it, but that's me.

Same story for the RBAC check down below as well.